src/test/java/org/takes/servlet/HttpServletResponseFakeTest.java

```diff
@@ -127,7 +138,7 @@ public void sendError() throws IOException {
         sresp.sendError(101, "Custom error message");
         MatcherAssert.assertThat(
             "Can't send a error in servlet response",
-            sresp.getHeaders("Status"),
+            sresp.getHeaders("HTTP/1.1"),
```

Comment:
@fanifieiev Why not use the constant here too?

Reply:
@paulodamaso Missed that, will fix.